core/types/log.go

```diff
@@ -70,18 +70,6 @@ type rlpLog struct {
 	Data    []byte
 }
 
-// legacyRlpStorageLog is the previous storage encoding of a log including some redundant fields.
```

Comment:
Can we add a description to `rlpLog` that: this struct is used to encode `consensus format` rlp and also for `storage format` rlp.